### ISSUE TITLE
Background for the Table on Gradebook Page

### ISSUE THREAD

**Issue by martinmts** [ksucpea/bettercanvas#3]
Hi,
Thanks for making and sharing the extension. It's great!

The table background on the Gradebook page is not in dark mode. The scores are less visible. Is it by design? Are there ways to make the table background in dark mode as well?

Thanks!

Dark mode enabled:
![image](https://user-images.githubusercontent.com/32575091/137423689-71d53ce6-1625-48a0-855b-3287cafd894c.png)

Compare with dark mode disabled:
![image](https://user-images.githubusercontent.com/32575091/137423754-759a5b54-58a0-4d6b-b78d-1986e1e73f92.png)

**Comment by ksucpea** [comment]
Whats the url for this page? I can't find it on my canvas

**Comment by martinmts** [comment]
It's `https://abc.instructure.com/courses/1234/gradebook`

**Comment by ksucpea** [comment]
Yeah I don't have access to that page so fixing this will be a little harder. Can you right click and go to "View page source" and copy paste everything and send that to my email ksucpea@gmail.com

**Comment by martinmts** [comment]
This page contains sensitive information. I will need to spend sometime to clear them out before sending it to you. I took a look at the source code. It does not seem to contain any student's information. I'm not sure if it helpful.

On the other hand, you may apply for a free _teacher_ account at https://canvas.instructure.com/login/canvas and have a better view of the page.

If you still need the source file, let me know. I'd be happy to clear the sensitive information and send it to you.
Thanks!
